marklogic-client-api/src/main/java/com/marklogic/client/extra/okhttpclient/OkHttpClientBuilderFactory.java

```diff
@@ -28,7 +28,7 @@
  */
 public interface OkHttpClientBuilderFactory {
 
-	static OkHttpClient.Builder newOkHttpClientBuilder(String host, int port, DatabaseClientFactory.SecurityContext securityContext) {
-		return OkHttpUtil.newOkHttpClientBuilder(host, port, securityContext);
+	static OkHttpClient.Builder newOkHttpClientBuilder(String host, DatabaseClientFactory.SecurityContext securityContext) {
```

Comment:
This is what ml-app-deployer users - it becomes simpler because the ML cloud auth configurer can safely assume usage of 443 for obtaining an access token, so "port" doesn't need to be provided anymore.